Traceback (most recent call last):
  File "analy_anova_plot.py", line 26, in <module>
    DATE = int(sys.argv[2])
ValueError: invalid literal for int() with base 10: '221014,'
